src/ctia/lib/es/index.clj

```diff
@@ -1,6 +1,8 @@
 (ns ctia.lib.es.index
   (:require
+   [clojure.core.memoize :as memo]
    [schema.core :as s]
+   [clj-time.core :as t]
```

Comment:
We should use ctia.lib.time as a facade to hide the time library implementation.